draft-ietf-jsonpath-base.md

```diff
@@ -1321,20 +1346,24 @@ Note that `..` on its own is not a valid segment.
 #### Semantics
 {: unnumbered}
 
-<!-- The following does not address https://github.com/ietf-wg-jsonpath/draft-ietf-jsonpath-base/issues/252 -->
-
 A descendant segment produces zero or more descendants of the input value.
 
-A nodelist enumerating the descendants is known as a _descendant nodelist_ when:
+A descendant selector of the form `..[<selectors>]` visits each node of the input value and
+its descendants in such an order that:
 
-* nodes of any array appear in array order,
-* nodes appear immediately before all their descendants.
+* nodes of any array are visited in array order, and
+* nodes are visited before all their descendants,
 
-This definition does not stipulate the order in which the children of an object appear, since
+applies the child segment `[<selectors>]` to each node, and concatenates the resultant nodelists together
```

Comment:
Was this meant to start a new sentence? It seems to start in the middle of one.

Reply:
No, but a sentence spanning the bullets was too sprawling, so I cut it in two. Thanks.